src/styles/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 4% auto;
```

Comment:
Don't add external styles to BEM block

Reply:
tell me please, what exactly is talking out

Reply:
tell me please, what exactly you talking about

Reply:
You can't add margins to BEM block. Read more [here](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en#setting-blocks-external-geometry-or-positioning)